app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -75,14 +77,22 @@ public StatsResult(SharedPreferences settings, long from, long to){
             avg = 0;
         }
 
+        if(getTotalReadings() > 0){
+            cursor= db.rawQuery("select ((count(*)*(sum(calculated_value * calculated_value)) - (sum(calculated_value)*sum(calculated_value)) )/((count(*)-1)*(count(*))) ) from bgreadings  where timestamp >= " + from + " AND timestamp <= " + to + " AND calculated_value > " + DBSearchUtil.CUTOFF + " AND snyced == 0", null);
+            cursor.moveToFirst();
+            stdev = cursor.getDouble(0);
+            stdev = Math.sqrt(stdev);
```

Comment:
StatsResult is used in Home.java and the widget that is recalculated quite often where we decided to not show the stdDev as it needs quite some time to calculate.

I did't see where you use it, but I think it would be better to move this to  a different utility class than the one that is used for Home and the Widget.